src/Nix/Options.hs

```diff
@@ -9,7 +9,7 @@ data Options = Options
     , tracing      :: Bool
     , thunks       :: Bool
     , values       :: Bool
-    , scopes       :: Bool
+    , showScopes   :: Bool
```

Comment:
@layus 
What do you think, should we rename CLI option from `scopes` to `showScopes`?
Since it is `Enable reporting of scopes in evaluation traces`, I think the `show*` verb is appropriate to add, because on itself `--scopes` says nothing what it does.
But if we rename it in that fashion - we would also align all other options to this idea.